### ISSUE TITLE
More info on sparse softmax appreciated

### ISSUE THREAD

**Issue by rish-16** [pyg-team/pytorch_geometric#4820]
### 📚 Describe the documentation issue

Hello PyG Team,

I've been trying to use PyG's **sparse softmax** operation (`tg.utils.softmax`) but am unable to understand what the input parameters are and what exactly is going on under the hood. I took a look at the source code but to no avail.

The `utils` ([link](https://pytorch-geometric.readthedocs.io/en/latest/modules/utils.html#)) page doesn't give any information about this softmax operation and there are no public tutorials either. When I try to click the method name, it has no hyper-link like the other `utils` methods do. 

Any help with this would be greatly appreciated! Thank you 🙌🏻

**Comment by rusty1s** [comment]
Thanks for reporting. There was indeed an issue in the code that prevented the `softmax` documentation from being displayed properly. This is now fixed (https://github.com/pyg-team/pytorch_geometric/pull/4824).
